agent/package.rb

```diff
@@ -45,6 +45,7 @@ class Package<RPC::Agent
         result = package_helper.apt_update
         reply[:exitcode] = result[:exitcode]
         reply[:output] = result[:output]
+        reply[:outdated_packages] = result[:outdated_packages]
```

Comment:
we will need to add `:outdated_packages` to the DDL as well - will work without but it helps a lot to have it.

Reply:
Good catch!  I amended the commit accordingly.